scoring/score.py

```diff
@@ -1,9 +1,57 @@
+DATA = {
+    # Location -> (Owner, Points if owner matches)
+    'arena':              (None, 0),
+    'docking_area':       (None, 1),
```

Comment:
Perhaps this should be a predicate?

Something like

```python
{
    'arena': lambda owner: 0,
    'docking_area': lambda owner: 1,
    'zone_0_raised_area': lambda owner: 3 if owner == 0 else 0,
    'zone_1_raised_area': lambda owner: 3 if owner == 1 else 0,
}
```

or

```python
{
    'arena': (0, lambda owner: True),
    'docking_area': (1, lambda owner: True),
    'zone_0_raised_area': (3, lambda owner: owner == 0),
    'zone_1_raised_area': (3, lambda owner: owner == 1),
}
```

Reply:
e83758cf8324194338de92dfe1d9b48aac7b904e moves to this style. I'm still not sure about it, though I think it does make the definition of this mapping clearer.